build.gradle

```diff
@@ -19,6 +19,7 @@ apply plugin: 'maven-publish'
 java {
     toolchain {
         languageVersion = JavaLanguageVersion.of(17)
+        vendor = JvmVendorSpec.BELLSOFT
```

Comment:
If we add this we will also want to add a plugin to settings.gradle.  See https://github.com/grails/grails-core/issues/13783

Reply:
I think I did that. Is there anything missing from settings.gradle?

Reply:
Sorry, I must have been looking at a stale version of the PR.  Looks great.